dev/fattest.simplicity/src/componenttest/topology/utils/MvnUtils.java

```diff
@@ -86,15 +85,20 @@ public static void init(LibertyServer server) throws Exception {
      */
     public static void init(LibertyServer server, String pomRelativePath) throws Exception {
         wlp = server.getInstallRoot();
-        home = new File(System.getProperty("user.dir"));
+        logDir = Props.getInstance().getFileProperty(Props.DIR_LOG);
+
         tckRunnerDir = new File("publish/tckRunner");
 
         pomXml = new File(tckRunnerDir, pomRelativePath);
 
         populateJarsFromWlp(pomXml);
 
-        mvnCliRaw = new String[] { "mvn", "clean", "test", "-Dwlp=" + wlp, "-Dtck_server=" + server.getServerName(),
-                                   "-Dtck_port=" + server.getPort(PortType.WC_defaulthost), "-DtargetDirectory=" + home.getAbsolutePath() + "/results/tck" };
+        String mvn = "mvn";
+        if (System.getProperty("os.name").contains("Windows")) {
+            mvn = mvn + ".cmd";
+        }
+        mvnCliRaw = new String[] { mvn, "clean", "test", "-Dwlp=" + wlp, "-Dtck_server=" + server.getServerName(),
+                                   "-Dtck_port=" + server.getPort(PortType.WC_defaulthost), "-DtargetDirectory=" + logDir.getAbsolutePath() + "/tck" };
```

Comment:
Is Props.DIR_LOG still under 'results' directoy or somewhere that is copied back to the main build results area from the child build?